src/sagemaker/amazon/amazon_estimator.py

```diff
@@ -93,8 +93,14 @@ def __init__(
             enable_network_isolation=enable_network_isolation,
             **kwargs
         )
-        data_location = data_location or "s3://{}/sagemaker-record-sets/".format(
-            self.sagemaker_session.default_bucket()
+
```

Comment:
Did you double check if the old data location also have a "/" in the end.?

Reply:
yep before in the case where the user didnt provide a data_location themselves, there would be a "/" at the end. lmk if Im misunderstanding the question